support/README.md

```diff
@@ -14,48 +15,95 @@ First response in an hour (during business time); updates to user until either s
-Any interesting new leads should be mentioned in #triage-customers. The
+These user support requests may lead to work that we need to track.
+
+We have two systems for tracking this work:
+
+- GitHub issues for all public, user-reported issues
+
+- Clubhouse for internal tracking and prioritization
+
+GitHub issues that we are actively working on should be tracked in
+Clubhouse. The reverse is not true - we have internal work
+that is not expressed in GitHub issues.
+
+### Support Triage
+
+When a user reports an issue, the Exterminator should ensure there's a Github issue.
+
+The exterminator should confirm with the user that the issue write-up accurately
+expresses their problem, and ask them to +1 it or comment with additional context.
+
+We use these votes and comments to decide which issues to prioritize. That
+_doesn't_ mean "the issue with the most votes wins". It's just one variable we
+use in prioritization.
+
+The exterminator should assign labels to the ticket that match our current
+work-tracking best practices. Issues with a bug label are automatically added to
+the [Tilt bug triage](https://github.com/orgs/windmilleng/projects/2).
+
+Any interesting new users should be mentioned in #triage-customers. The
 exterminator should use their best judgment on who is promising enough to DM
 directly (based on a combination of research on their background and how active
 they are in the channel).
 
-The exterminator is responsible for making sure loops get closed,
-non-exterminators can reply but should tend to defer.
+### Internal Support Triage
 
-The exterminator is also responsible for making sure a
-[release](https://github.com/windmilleng/tilt/blob/master/DEVELOPING.md#releasing)
-happens every non-holiday week.
+When a Tilter has a problem with Tilt, they should report it in the #triage-bugs Slack
+channel.
 
-## Bug Priority
+The exterminator should determine whether the problem belongs in Github Issues or
+Clubhouse, based on whether users have noticed and who it affects, and file it
+in the appropriate place.
 
-The exterminator picks which bugs are high-priority to deserve immediate attention. Factors include:
+## Request Priority
+
+The exterminator picks which support requests are high-priority to deserve
+immediate attention. Factors include:
 
 - Is there a workaround? (If so, the exterminator owns verifying the user is able to use it)
 
 - How bad is the impact? Are there side effects (like data loss in the cluster)
 
 - How common is it?
 
-- Is it blocking a high-priority customer?
+- Is it blocking a high-priority team?
+
+If you aren't sure, ask a teammate or in #triage-bugs.
+
+For high priority requests, the exterminator should assign someone to start work
+immediately.
+
+Low priority requests should be tracked in Clubhouse and teams should make their own
+judgements about how to prioritize them. If the exterminator sees a low priority
+request that they know how to fix quickly, they have the authority to take action.
```

Comment:
```suggestion
request that they know how to fix quickly, they should consider fixing it right away.
```

I don’t like the word “authority”. If mis-understood, one may think Tilt engineers don’t have authority to take action under general circumstances. I’d like to think Tilters should have freedom to pick what to work on, guide by documents like this to make good judgement.

Reply:
done!